scala/README.md

```diff
@@ -644,14 +669,17 @@ Tips & Tricks
 
 0. You can use the `@scala.beans.BeanProperty` and `@BooleanBeanProperty` annotations to automatically generate JavaBeans style getter and setter methods.
 
-0. Use this syntax to pass a sequence as a parameter to a variable length argument list method:
+0. If a method takes a parameter of type `Seq[A]`, put that parameter last and make it a variable-length argument (_vararg_). Use the syntax below to pass a sequence as a parameter to the method, if needed:
```

Comment:
i disagree that this should be recommended "if a method takes a paramater of type `Seq[A]`". in my experience varargs is only helpful in order to make the interface for some method nice to use (often when building DSL's or the outer interface of a library). otherwise it just gives you an extra step you have to do to pass lists to functions (`list: _*`).